.changeset/good-mirrors-bake.md

```diff
@@ -0,0 +1,7 @@
+---
+'astro': minor
+---
+
+Improved performance of optimized images generation at build time. Astro will now generate optimized images concurrently, which can significantly speed up build times for sites with many images. Additionally, Astro will now reuse the same buffer for all variants of an image, which should improve performance for sites websites with many variants of the same image, especially when using remote images.
```

Comment:
```suggestion
Improved image optimization performance

Astro will now generate optimized images concurrently at build time, which can significantly speed up build times for sites with many images. Additionally, Astro will now reuse the same buffer for all variants of an image. This should improve performance for websites with many variants of the same image, especially when using remote images.
```
For the minor, just made the first line closer to a blog headline! Something like that maybe?

Reply:
This looks great to me!